ts/deploy.ts

```diff
@@ -256,18 +253,3 @@ export async function deployAll(
         withdrawManager
     };
 }
-
-async function getMerkleRootWithCoordinatorAccount(
-    parameters: DeploymentParameters
-) {
-    const state0 = State.new(0, 0, 0, 0);
-    const state1 = State.new(1, 0, 0, 0);
-    const dataLeaves = [state0.toStateLeaf(), state1.toStateLeaf()];
-    const ZERO_BYTES32 =
-        "0x290decd9548b62a8d60345a988386fc84ba6bc95484008f6362f93160ef3e563";
-    for (let i = dataLeaves.length; i < 2 ** parameters.MAX_DEPTH; i++) {
-        dataLeaves[i] = ZERO_BYTES32;
-    }
```

Comment:
Building the 32 levels of the leaves is why the deploy script freezes a second before deploying the rollup contract.